src/test/lib/lit-extended_test.ts

```diff
@@ -203,6 +203,36 @@ suite('lit-extended', () => {
       assert.equal(count2, 1);
     });
 
+    test('allows updating event listener without extra calls to remove/addEventListener', () => {
+      let listener: Function|null;
+      const t = () => html`<div on-click=${listener}></div>`;
```

Comment:
I think the test would be a little easier to follow if the listener was just a parameter to `t` rather than a variable.